app/shared/containers/Tools.js

```diff
@@ -58,7 +58,7 @@ const paneMapping = [
   },
   {
     element: ToolsWallets,
-    modes: ['cold', 'hot', 'ledger', 'watch'],
+    modes: ['cold', 'hot', 'ledger', 'watch', 'skip'],
```

Comment:
@aaroncox could it be that you had skipped the setup the one time that the "Manage Wallets" section was hidden for you? Maybe that should show up in the sidebar even when the setup was skipped?

Reply:
Could be! We can def test to see if this resolves it